includes/Core/Modules/Module_With_Assets_Trait.php

```diff
@@ -0,0 +1,73 @@
+<?php
+/**
+ * Trait Google\Site_Kit\Core\Modules\Module_With_Assets_Trait
+ *
+ * @package   Google\Site_Kit
+ * @copyright 2020 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Modules;
+
+use Google\Site_Kit\Core\Assets\Asset;
+
+/**
+ * Trait for a module that includes assets.
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+trait Module_With_Assets_Trait {
+
+	/**
+	 * List of the module's Asset objects to register.
+	 *
+	 * @since n.e.x.t
+	 * @var array
+	 */
+	protected $registerable_assets;
```

Comment:
Why not simply `$assets` since it is used to memoize the return value  of `get_assets`?

Reply:
I wanted to use a more distinct name to avoid a weird collision, should we ever want to add `Assets` awareness to a `Module`. It's mostly a property internal to the trait anyway, so I think that's okay.